.github/workflows/build-test-core-x86-ocaml5.jsonnet

```diff
@@ -6,8 +6,7 @@ local actions = import 'libs/actions.libsonnet';
 local semgrep = import 'libs/semgrep.libsonnet';
 
 //TODO: currently fails because of conflicts with ocamlformat and jsoo
-//TODO: local opam_switch = '5.2.0~alpha1';
-local opam_switch = '5.1.0';
+local opam_switch = 'ocaml-base-compiler.5.2.0~beta2';
```

Comment:
Add a comment explaining why we can't use 5.2.0~beta2; just add a link to the issue

Reply:
will do